src/cmake/linux.cmake

```diff
@@ -53,11 +53,6 @@ if(NOT BUILD_FLATPAK)
     target_sources(mozillavpn PRIVATE
         ${CMAKE_SOURCE_DIR}/3rdparty/wireguard-tools/contrib/embeddable-wg-library/wireguard.c
         ${CMAKE_SOURCE_DIR}/3rdparty/wireguard-tools/contrib/embeddable-wg-library/wireguard.h
-        ${CMAKE_SOURCE_DIR}/src/daemon/daemon.cpp
-        ${CMAKE_SOURCE_DIR}/src/daemon/daemon.h
-        ${CMAKE_SOURCE_DIR}/src/daemon/dnsutils.h
-        ${CMAKE_SOURCE_DIR}/src/daemon/iputils.h
-        ${CMAKE_SOURCE_DIR}/src/daemon/wireguardutils.h
```

Comment:
This is because it's also in the shared sources cmake file, right?

Reply:
That is correct, this is just some cleanup that I missed in #9584 and I stumbled across it while working on this PR.